malariagen_data/anopheles.py

```diff
@@ -6876,6 +7062,320 @@ def plot_g123_calibration(
+        width: gplt_params.width = gplt_params.width_default,
+        height: gplt_params.height = 200,
+        show: gplt_params.show = True,
+        x_range: Optional[gplt_params.x_range] = None,
+    ):
+        # compute ihs
+        x, xpehh = self.xpehh_gwss(
+            contig=contig,
+            analysis=analysis,
+            window_size=window_size,
+            percentiles=percentiles,
+            standardize=standardize,
+            standardization_bins=standardization_bins,
+            standardization_n_bins=standardization_n_bins,
+            standardization_diagnostics=standardization_diagnostics,
+            filter_min_maf=filter_min_maf,
+            map_pos=map_pos,
+            min_ehh=min_ehh,
+            max_gap=max_gap,
+            gap_scale=gap_scale,
+            include_edges=include_edges,
+            use_threads=use_threads,
+            min_cohort_size=min_cohort_size,
+            max_cohort_size=max_cohort_size,
+            sample_query1=sample_query1,
+            sample_query2=sample_query2,
+            sample_sets=sample_sets,
+            random_seed=random_seed,
+        )
+
+        # determine X axis range
+        x_min = x[0]
+        x_max = x[-1]
+        if x_range is None:
+            x_range = bokeh.models.Range1d(x_min, x_max, bounds="auto")
+
+        # create a figure
+        xwheel_zoom = bokeh.models.WheelZoomTool(
+            dimensions="width", maintain_focus=False
+        )
+        if title is None:
+            if sample_query1 is None or sample_query2 is None:
+                title = "XP-EHH"
+            else:
+                title = "'" + sample_query1 + "' and '" + sample_query2 + "'"
+        fig = bokeh.plotting.figure(
+            title=title,
+            tools=["xpan", "xzoom_in", "xzoom_out", xwheel_zoom, "reset"],
+            active_scroll=xwheel_zoom,
+            active_drag="xpan",
+            sizing_mode=sizing_mode,
+            width=width,
+            height=height,
+            toolbar_location="above",
+            x_range=x_range,
+        )
+
+        if window_size:
+            if isinstance(percentiles, int):
+                percentiles = (percentiles,)
+
+        # add an empty dimension to xpehh array if 1D
+        xpehh = np.reshape(xpehh, (xpehh.shape[0], -1))
+        bokeh_palette = bokeh.palettes.all_palettes[palette]
+        for i in range(xpehh.shape[1]):
+            xpehh_perc = xpehh[:, i]
+            if xpehh.shape[1] >= 3:
+                color = bokeh_palette[xpehh.shape[1]][i]
+            elif xpehh.shape[1] == 2:
+                color = bokeh_palette[3][i]
+            else:
+                color = None
+
+            # plot ihs
+            fig.circle(
+                x=x,
+                y=xpehh_perc,
+                size=3,
+                line_width=0.15,
+                line_color="black",
+                fill_color=color,
+            )
```

Comment:
```suggestion
            fig.circle(
                x=x,
                y=xpehh_perc,
                size=4,
                line_width=0,
                line_color=color,
                fill_color=color,
            )
```

Looks better with webgl.